specs/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../.eslintrc.json",
+  "env": {
+    "jasmine": true
+  },
+  "rules": {
+    "no-unused-vars": "off"
```

Comment:
I copied it from https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/288/files/30d340f9d3c9b7d222f59431bd1e11a5f078c952#diff-6deaf0988d1fe3f5d08d3f8bca32b624L4. Want me to turn it off and remove unused vars?